tapestry.js

```diff
@@ -2982,4 +2977,23 @@ $(document).ready(function(){
         var y = event.clientY + $(window).scrollTop();
         recordAnalyticsEvent('user', 'click', 'tapestry', null, {'x': x, 'y': y});
     }, true);
-});
\ No newline at end of file
+});
+
+function getDomNode(idOrClass){
+    const isClass = idOrClass[0] === ".";
+    idOrClass = idOrClass.substring(1);
+    return isClass ? document.getElementByClassName(idOrClass) : document.getElementById(idOrClass);
+}
+
+function removeParentNode(idOrClass) {
+    let oNode = getDomNode(idOrClass);
+    oNode.parentId.removeChild(oNode);
```

Comment:
What is `parentId`?

Reply:
this is a suggestion from [http://youmightnotneedjquery.com/#remove](http://youmightnotneedjquery.com/#remove)